app/gui/src/presenter/graph/visualization/manager.rs

```diff
@@ -94,6 +75,7 @@ pub enum Notification {
 
 /// Describes the state of the visualization on the Language Server.
 #[derive(Clone, Debug, PartialEq)]
+#[allow(clippy::large_enum_variant)]
```

Comment:
Clippy complains about the size of variant enum. I though that it is safe to ignore because most of the variants have comparable sizes (the size of `Visualization` struct). And the variant having two `Visualization` fields is the one that Clippy complains about.